backend/pick-git/src/test/java/com/woowacourse/pickgit/post/domain/PostRepositoryTest.java

```diff
@@ -58,12 +72,34 @@ void setUp() {
     void save_SavedPost_Success() {
         // given
         Images images = new Images(List.of(new Image(image)));
-        Post post = new Post(postContent, images, user);
+        Post post = new Post(postContent, images, githubRepoUrl, user);
 
         // when
         Post savedPost = postRepository.save(post);
 
         // then
         assertThat(savedPost.getId()).isNotNull();
     }
+
+    @DisplayName("게시글을 저장할 때 태그도 함께 영속화된다.")
+    @Test
+    void save_WhenSavingPost_TagSavedTogether() {
+        Post post =
+            new Post(null, null, new PostContent(), githubRepoUrl, null, null, new ArrayList<>(), null);
+        List<Tag> tags = Arrays.asList(new Tag("tag1"), new Tag("tag2"));
+        post.addTags(tags);
+        postRepository.save(post);
+        Tag entityTag = tagRepository.save(new Tag("33"));
+        post.addTags(Arrays.asList(entityTag));
+
+        testEntityManager.flush();
+        testEntityManager.clear();
+
+
+        Post findPost = postRepository.findAll()
+            .get(0);
```

Comment:
```suggestion
        Post findPost = postRepository.findAll().get(0);
```

이 부분은 개행 없어도 될 거 같아요!